src/CommunityToolkit.Maui.Markup/CommunityToolkit.Maui.Markup.csproj

```diff
@@ -1,41 +1,49 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-	<PropertyGroup>
-		<TargetFramework>net6.0</TargetFramework>
-		<UseMaui>true</UseMaui>
-		<GenerateDocumentationFile>true</GenerateDocumentationFile>
-	</PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
```

Comment:
Was this required to fix the `error MSB4057: The target "CopyLocalFilesOutputGroup` bug? 

It shouldn't be necessary; `CommunityToolkit.Maui.Markup-rc2` works on Windows.

I'd prefer to only target `net6.0` since `Maui.Markup` doesn't require any platform-specific code.

Reply:
yes, because this is a Maui library, it needs to include all target frameworks. Hope MAUI team will fix it, but for now as you can see it is required

Reply:
Interesting that `CommunityToolkit.Maui.Markup` works fine on Windows when used as a NuGet Package, but throws a compiler error when referenced as a project.

I think the solution then is to revert these changes, keeping `net6.0` a the only TFM for `CommunityToolkit.Maui.Markup`, and remove the windows TFM from `CommunityToolkit.Maui.Markup.Sample` until the .NET MAUI resolves the bug: https://github.com/dotnet/maui/issues/6529

Reply:
Then we'll re-add the windows TFM to the sample app once the bug is fixed